contrib/wlroots-portals.conf

```diff
@@ -0,0 +1,7 @@
+[preferred]
+# Use xdg-desktop-portal-gtk for every portal interface...
+default=gtk
+# ... except for the Screencast, Screenshot and Settings (dark/light mode) interface
+org.freedesktop.impl.portal.Screencast=wlr
```

Comment:
`ScreenCast`.

The case mismatch causes x-d-p to fallback to UseIn, which inconveniently hides the error. But at least there's a log:
```
xdg-desktop-por[24921]: Choosing wlr.portal for org.freedesktop.impl.portal.ScreenCast via the deprecated UseIn key
xdg-desktop-por[24921]: The preferred method to match portal implementations to desktop environments is to use the portals.conf(5) configuration file
```

Reply:
Seems like this might also be an issue upstream?

https://github.com/flatpak/xdg-desktop-portal/pull/955/files#diff-0958e51ba7fb0f74e91c3e294229b576a1e7d022c734f45aaa10c5a56b72c268R72